tests/documentation/test_files/DocumentationStyleBear/bad_file.py.test.correct

```diff
@@ -1,4 +1,5 @@
 def improper_indents(indents):
+
```

Comment:
That comment that you link points to PEP 257, and quotes from it, showing why this is wrong.

I accepted that the bear didnt enforce that aspect of PEP257 because it was the first merge of the bear, which would have to improve later.
And now your project is here, and you *are* enforcing a PEP257 violation.